### DISCUSSION TITLE
Flycast lose sound

### DISCUSSION THREAD

**Discussion by BaronOfLight** [flyinghead/flycast#984] [Q&A] [answered]
Very new to github and Dreamcast Emulation here, so please talk to me as if I have no idea what is what.

I have a problem where whenever I entered Flycast emulation setting during gameplay, I will lose sound when I go back to gameplay. Whether its to change control mapping, or load a state from a prev play, the sound will lose. So far the games that I have trouble with these are Cyber Troopers Virtual On Oratorio Tangram(US), Tech Romancer(US), Phantasy Star Online Ver2(US), Zero Gunner 2 (JP).

My expertise in manipulating setting in any emulators are just changing control/mapping control, save and load states, and cheat codes. GFX settings, emulation settings, Sound settings are beyond my understanding. I just let those be. Dreamcast emulation is very new to me. Past emulators that I have used were for PSX, PS2, PSP, GBA, NES and SNES.

**Comment by flyinghead** [comment, marked as answer]
It is very likely an issue with your audio card or driver. When you pause a game, Flycast completely stops audio, and re-initializes it when resuming the game.
Check your audio settings, upgrade your audio driver and disable/unplug any audio component or device that you don't need.

Also try to use a different audio driver in Flycast (Settings > Audio).

**Reply by BaronOfLight** [reply]
Afai can tell, it worked. Well, I changed Flycast Audio setting to directsound-Microsoft Directsound. My own PC's setting... I have no idea what does what. It is a very old machine. Running Win7.
